lib/zendesk_messaging.dart

```diff
@@ -139,6 +143,30 @@ class ZendeskMessaging {
     return completer.future;
   }
 
+  /// Retrieve uread messages count from the Zendesk SDK
+  static Future<int> messageCount() async {
```

Comment:
rename, thanks.

Reply:
Fixed in 20b802ed8f51d865d4df359d352885757e5c5a9c